lib/graphql/execution/interpreter/handles_raw_value.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module GraphQL
+  module Execution
+    class Interpreter
+      # Wrapper for raw values
+      class RawValue
+        attr_reader :object
+
+        def initialize(obj = nil)
+          @object = obj
+        end
+
+        alias_method :resolve, :object
```

Comment:
Why have an alias for this method?

```suggestion
```

Reply:
The idea is to use `resolve` everywhere for consistency.
We assume that users/libraries would be able to sub-class RawValue and implement their own resolving logic.

For example, CachedValue could be smth like:

```ruby
class CachedValue < GraphQL::Execution::Interpreter::RawValue
  def resolve
     cache_store.read(object.cache_key) || object
  end
end
```

Reply:
Thanks for clarifying, that makes sense to me 👌  

In that case, maybe we can get rid of `attr_reader :object` ?

Reply:
Agreed, removed it